### ISSUE TITLE
If a node that's been provisioned, then deleted, is restarted...it does not get reprovisioned

### ISSUE THREAD

**Issue by jameskyle** [att-innovate/occam#26]
For example:

```
vagrant up ctrl1
# node is provisioned correctly
vagrant destroy ctrl1
vagrant up ctrl
# node is now registered as deployed and sent to boot from disk, which fails.
```

This bug would also appear on bare metal if a system, say, was reformatted or lost a root disk but whose nics (and macs) remained the same.

May be related to #25

**Comment by doctorjnupe** [comment]
I think that this is the desired functionality.  We should have another process that allows you to clear a registered node from the ops node and redeploy the node.  We've done this before but it was a manual process.  I did build a rake task that reset razor but that has become obsolete with the invention of the occamengine.

**Comment by jameskyle** [comment]
Yeah, I didn't mean to imply that vagrant or occam _should_ reprovision but that we ought to figure out a work flow that doesn't require blowing away the whole environment.

That's what I meant by "related to #25".
